e2e/cypress.env.json

```diff
@@ -0,0 +1,3 @@
+{
+    "setChromeWebSecurity": false
+}
```

Comment:
How were the saml e2e tests working without this setting before?

Reply:
This setting is to be set before running these tests. This file should not be checked in, thanks for catching this.

Reply:
Removed...